services/121-service/src/registration/services/registrations-import.service.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { TestBed } from '@automock/jest';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { ProgramEntity } from '../../programs/program.entity';
+import { UserService } from '../../user/user.service';
+import { LanguageEnum } from '../enum/language.enum';
+import { RegistrationsImportService } from './registrations-import.service';
+
+describe('RegistrationsImportService', () => {
+  let registrationsImportService: RegistrationsImportService;
+
+  const programId = 2;
+  const language = LanguageEnum.en;
+  const importRegistrationsCsvInput = [
+    {
+      namePartnerOrganization: 'ABC',
+      preferredLanguage: language,
+      paymentAmountMultiplier: '',
+      maxPayments: '5',
+      nameFirst: 'Test',
+      nameLast: 'Test',
+      phoneNumber: '31600000000',
+      fspName: 'Intersolve-voucher-paper',
```

Comment:
There's am Enum for this value as well, right?

Reply:
done